content/en/docs/best-practices/common.md

```diff
@@ -17,6 +17,12 @@ Check and test your operator bundle before you publish it. Note that the [`opera
 operator-sdk bundle validate ./bundle --select-optional suite=operatorframework
 ```
 
+Note that some validators can allow performing the check against the Kubernetes version where that is intend to be published:   
```

Comment:
Lets be specific about this:
```suggestion
The OperatorHub.io validator in the `operatorframework` optional suite allows you to validate that your manifests can work with a Kubernetes cluster of a particular version using the `k8s` optional value:
```

Reply:
My intention here is to keep that generic. So, if new validators allow that this doc will not get outdated. 

IHMO the details over what validator and what options are available for each case requires to get done in the API docs https://github.com/operator-framework/api/issues/109. Then, we could link that to other places for further details.